assets/styles/style.css

```diff
@@ -437,10 +420,31 @@ PLUGIN STYLES
   margin: 0 auto;
 }
 
+.header-grid {
+  display: grid;
+  grid-template-columns: 40px 1fr 40px;
+  grid-template-rows: 40px 40px;
+  grid-template-areas: 'Logo TopMenu User' 'SubMenu SubMenu SubMenu';
+  margin: 10px 15px;
+  border-bottom: 1px solid #ccc;
+  padding-bottom: 30px;
+ }
+
 @media (min-width:768px) {
   .header-grid {
-      grid-template-columns: 60px 1fr 60px;
-      grid-template-rows: 30px 30px;
-      grid-template-areas: 'Logo TopMenu User' 'Logo SubMenu User';
+    grid-template-columns: 60px 1fr 60px;
+    grid-template-rows: 30px 30px;
+    grid-template-areas: 'Logo TopMenu User' 'Logo SubMenu User';
+    border-bottom: 1px solid #ccc;
```

Comment:
The `border-bottom` and `padding-bottom` here are identical to the ones in `.header-grid` outside of the media query, so there's no need to have them here; they'll just get picked up anyway. Please remove them as they are redundant.